### ISSUE TITLE
For videos that change rapidly

### ISSUE THREAD

**Issue by SmileShaun** [qiuyu96/CoDeF#20]
it's really a very good job, but i have some problems.I want to do a video translation of a dancing video, but the canonical image is very bad, as follows:
![canonical_0 (1)](https://github.com/qiuyu96/CoDeF/assets/118582474/af119a64-25a2-4b55-97ce-16b0dad32542)
I wonder if my hyperparameters are set incorrectly? Or is it slightly less effective for videos with a faster rate of change?

**Comment by SmileShaun** [comment]
> Hi, thank you for your attention. Your question is closely related to issue [#2](https://github.com/qiuyu96/CoDeF/issues/2). We have provided a detailed reply [here](https://github.com/qiuyu96/CoDeF/issues/2#issuecomment-1680109813).

Thanks for your reply, I guess you probably didn't understand me. The duration of my video is not long, but the movements of the characters are larger and the video changes more rapidly. So it's a bit different from issue [#2](https://github.com/qiuyu96/CoDeF/issues/2).

**Comment by qiuyu96** [comment]
Our method has limitations in specific extreme cases, as discussed in Section 5 of our [paper](https://arxiv.org/pdf/2308.07926.pdf). However, you can still try to adjust some key hyperparameters, including but not limited to:
* The size of the hash table.
* The depth and width of the MLP.
* The maximum training time.
* The annealing speed. 
* Learning rate of the deformation MLP and canonical MLP, respectively.


Hope it will be helpful to you.

**Comment by SmileShaun** [comment]
> Our method has limitations in specific extreme cases, as discussed in Section 5 of our [paper](https://arxiv.org/pdf/2308.07926.pdf). However, you can still try to adjust some key hyperparameters, including but not limited to:
> 
> * The size of the hash table.
> * The depth and width of the MLP.
> * The maximum training time.
> * The annealing speed.
> * Learning rate of the deformation MLP and canonical MLP, respectively.
> 
> Hope it will be helpful to you.

Thank you very much for your answer

**Comment by Jackylee2032** [comment]
> 

Please my qq981687874
